Package/TextMate Preferences/TextMate Preferences.sublime-syntax

```diff
@@ -1,538 +1,742 @@
 %YAML 1.2
 ---
 name: TextMate Preferences (PList / XML)
+scope: text.xml.plist.textmate.preferences
+version: 2
+
+extends: Packages/PackageDev/Package/Property List/Property List.sublime-syntax
```

Comment:
Scope selector syntaxes live next to each other, so it's easy to use relative paths. I have onestly not tried whether something like `../Property List/Property List.sublime-syntax` works, so used a path I know it works.

Reply:
I think if only the basename is provided, ST will perform its usual resource merging algorithm which may end up with us trying to extend the wrong syntax if it was loaded in a later package (such as "User"). Using the full paths here is fine and preferred.

Reply:
Not sure about it, as relative extends are limited to syntaxes within the same package. In general `extends` is designed to be feeded with absolute imports to avoid ambiguities.